elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreResourceDefinition.java

```diff
@@ -192,7 +216,17 @@ private CredentialStoreAddHandler() {
         protected void performRuntime(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
 
             ModelNode model = resource.getModel();
-            String uri = asStringIfDefined(context, URI, model);
+            String location = asStringIfDefined(context, LOCATION, model);
+            boolean modifiable = asBooleanIfDefined(context, MODIFIABLE, model);
```

Comment:
If not defined, this method seems to return `false` while the default value for the attribute  is `true`. `true` should be used if not defined.